package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "tslint-config-seek",
+  "version": "0.0.0-development",
+  "description": "TSLint configuration used by SEEK",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/seek-oss/tslint-config-seek.git"
+  },
+  "author": "SEEK",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/seek-oss/tslint-config-seek/issues"
+  },
+  "scripts": {
+    "commit": "git-cz",
+    "travis-deploy-once": "travis-deploy-once",
```

Comment:
Given we are only running our pipeline on one version of node we can remove the need for `travis-deploy-once`